Mathlib/Data/List/EditDistance/Defs.lean

```diff
@@ -116,8 +116,8 @@ theorem impl_cons (w' : 0 < List.length ds) :
 
 -- Note this lemma has two unspecified proofs: `h` appears on the left-hand-side
```

Comment:
Can you update this comment as well?

Reply:
There's no need to update: the proofs are still there and will generate side goals when rewriting. All that has changed is I've written the types of the proofs.